frontend/src/components/layouts/AnalyticsNavigation.tsx

```diff
@@ -67,10 +69,17 @@ const AnalyticsNavigation: React.FC<INavigationProps> = (
     },
   ];
 
+  const getNavItems = () => {
+    if (currentUser.role === IUserRole.ANALYST_READER) {
+      return [...analyticsNavItems.slice(0, 2), ...analyticsNavItems.slice(3)];
```

Comment:
try `array.slice().splice(2, 1)`

Reply:
```
const array = [0, 1, 2, 3];
console.log([... array.slice(0, 2), ... array.slice(3)]);
>> [0, 1, 3]
console.log(array.slice().splice(2, 1));
>> [2]
```

Reply:
`[... array.slice(0, 2), ... array.slice(3)]` It's what I want